addons/zotero/views.py

```diff
@@ -57,12 +64,80 @@ def library_list(self):
+                    requested_resource=project,
+                    params={'filter[resource_uri]': project_ulr},
+                )
+                configured_citation_addons_url = resource_references_response.get_related_link(
+                    'configured_citation_addons')
+                addons_url_list = get_gv_result_json(
+                    endpoint_url=configured_citation_addons_url,
+                    requesting_user=auth.user,
+                    requested_resource=project,
+                    request_method='GET',
+                    params={}
+                )
+                gv_addon_name = addon_short_name if addon_short_name != 'zotero' else 'zotero_org'
+                citation_url_list = list(
+                    filter(lambda x: x['attributes']['external_service_name'] == gv_addon_name, addons_url_list))
+                for addon in citation_url_list:
+                    gv_response = _make_gv_request(
+                        endpoint_url=f'{settings.GRAVYVALET_URL}/v1/addon-operation-invocations/',
+                        requesting_user=auth.user,
+                        requested_resource=project,
+                        request_method='POST',
+                        params={},
+                        data={
+                            'data': {
+                                'attributes': {
+                                    'operation_name': 'list_root_items',
+                                    'operation_kwargs': {},
+                                },
+                                'relationships': {
+                                    'thru_addon': {
+                                        'data': {
+                                            'type': addon['type'],
+                                            'id': addon['id']
+                                        }
+                                    }
+                                },
+                                'type': 'addon-operation-invocations'
+                            }
+                        }
+
+                    )
+                    if gv_response.status_code == 201:
+                        attributes_dict = gv_response.json()['data']['attributes']
+                        items = attributes_dict.get('operation_result').get('items')
+                        for item in items:
+                            item_id = item.get('item_id', '')
+                            response.append({
+                                'addon': addon_short_name,
+                                'kind': item.get('item_type', '').lower(),
+                                'id': item_id,
+                                'name': item.get('item_name', ''),
+                                'path': item.get('item_path', ''),
+                                'parent_list_id': None,
+                                'provider_list_id': item_id,
+                            })
+                return response
```

Comment:
AFAIK this is only used during configuration of Zotero addon in legacy way, and therefore we should not need this.Please confirm whether we really need this, and if so, please reuse `citation_list_gv_request` function